internal/pkg/template/templates/workloads/services/static-site/manifest.yml

```diff
@@ -16,19 +16,9 @@ files:
 {{- else}}
 files:
 {{- range $file := .FileUploads}}
-  - source: {{$.Source}}
-    destination: {{$.Destination}}
-    {{- if $.Context}}
-    context: {{$.Context}}
-    {{- end}}
-    {{- if $.Recursive}}
-    recursive: {{$.Recursive}}
-    {{- end}}
-    {{- if $.Reinclude.StringSlice}}
-    reinclude: {{$.Reinclude.StringSlice}}
-    {{- end}}
-    {{- if $.Exclude.StringSlice}}
-    exclude: {{$.Exclude.StringSlice}}
+  - source: {{$file.Source}}
```

Comment:
hmm do we not write destination?

Reply:
added to be the same as the case without specified sources. wondering what our manifest validation should look like for `destination`....